exchange/bidder.go

```diff
@@ -389,7 +389,15 @@ func (bidder *bidderAdapter) doTimeoutNotification(timeoutBidder adapters.Timeou
 			}
 		} else {
 			bidder.me.RecordTimeoutNotice(false)
+			if bidder.DebugConfig.TimeoutNotification.Log {
+				msg := fmt.Sprintf("TimeoutNotification: Failed to make timeout request: method(%s), uri(%s), error(%s)", toReq.Method, toReq.Uri, err.Error())
+				util.LogRandomSample(msg, glog.Warningf, bidder.DebugConfig.TimeoutNotification.SamplingRate)
+			}
 		}
+	} else if bidder.DebugConfig.TimeoutNotification.Log {
+		reqJSON, _ := json.Marshal(req)
```

Comment:
Should a metric be added here as well for an error in generating the timeout notification event request?

Reply:
I am a little wary of creating too many metrics here, especially if we may move to per adapter metrics in the future. Do you think the same failure metric would make sense here?